Sources/ClientRuntime/Orchestrator/OrchestratorTelemetry.swift

```diff
@@ -0,0 +1,117 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import class Smithy.Context
+import struct Smithy.Attributes
+import struct Smithy.AttributeKey
+import enum SmithyHTTPAPI.SmithyHTTPAPIKeys
+
+/// Container for Orchestrator telemetry, including configurable attributes and names.
+///
+/// Note: This is intended to be used within generated code, not directly.
+public class OrchestratorTelemetry {
```

Comment:
`OrchestratorTelemetry` is a container for all the state (e.g. attributes) and meters (used for metrics) for a given `Orchestrator` instance. The container does not emit any metrics itself, and only provides convenient access standardized metrics.

Metrics captured:

- smithy.client.call.duration
- smithy.client.call.attempts
- smithy.client.call.errors
- smithy.client.call.attempt_duration
- smithy.client.call.resolve_endpoint_duration
- smithy.client.call.serialization_duration
- smithy.client.call.deserialization_duration
- smithy.client.call.auth.signing_duration
- smithy.client.call.auth.resolve_identity_duration